Adding call to action

clcoffey opened the pull request:
Adding call to action on the home page pointing to the State of InnerSource Survey.

rrrutledge commented on resources/learningpath/index.md:
```diff
@@ -12,31 +12,36 @@ We hope they help you on your journey to InnerSource!
 
 Check out the Introduction to learn about the types of problems where InnerSource can help, how to do it, the types of benefits you can expect to see by participating, and the underlying principles that make it all work.
+
+View in [English][Introduction], [German][German Introduction], [Japanese][Japanese Introduction], or [Chinese][Chinese Introduction]
```
@lenucksi what word shall we use to represent German?  `Deutsche`?

lenucksi replied:
You were 99% on spot, it should be `Deutsch` and mostly `Deutsche` are going to read it. :wink:

tsadler1988 reviewed (approved):
Great work Russ!

I think this is good to merge as is, but some ideas for further improvements:
- Translate 'Back to Learning Path homepage', 'Next', and 'Previous'
- Would linking to the English workbooks be better than nothing?
- Would embedding the English video be better than nothing?

rrrutledge commented:
> some ideas for further improvements

Awesome!  Thanks for this review.  Can you open and issue or otherwise bring this up in the #learning-path.

tsadler1988 commented:
> Awesome! Thanks for this review. Can you open and issue or otherwise bring this up in the #learning-path.

Late, but done!
- https://github.com/InnerSourceCommons/InnerSourceLearningPath/issues/331
- https://github.com/InnerSourceCommons/InnerSourceLearningPath/issues/332
- https://github.com/InnerSourceCommons/InnerSourceLearningPath/issues/333

rrrutledge commented:
Thanks, Tom!

clcoffey committed:
Adding call to action

Adding call to action on the home page pointing to the State of InnerSource Survey.

claredillon reviewed (approved):
Thanks for this change, Cristina. 

Noting it was prompted by @kjstol 's suggestion to experiment with a more graphical Call to Action on the front page of the website. 

We have also discussed changing it to a more general Call to Action (e.g. sign up to newsletter) after the survey finished.
